examples/plotting_with_folium.ipynb

```diff
@@ -0,0 +1,457 @@
a4OW4adm5tQr5zasAm1hj+P1+JBIJCIKA6+vrrmuO1WpFJBJBMBicWQRGrVbDly9fxn5fMBiE3W6Hw+GA3W6n+gzEq4POeIIgFkK73V6rqANiNRjHGbuMisXhcPhF57kRxDwJBoPIZrNzzypnfQIr5mXMpuZ5XhcnBolGoVDIVJQJx3EjX9dut3Fzc4NGo4F3797pjyeTSciy3LXEcxC1Wg339/eIx+MjhVO73Y6joyOUSiVks1ld2DP2q0xsYtEI7PneYl+TuGZ7cTgc8Pv9U21jUbDiYtFoFOl0Gnd3d13L94370OhWZi7lUcusc7kc/H6/acc18CSkxONxxONxKIqCh4cHpNNp0+83kz3M8/xcxUdN0/R9NA2srb1CKmt7r+jIxFlg/PiFfr+FdRc0e2FCLRP1es8VJloPOofY+xYdL8DEd+DHifrXDNv37JztdDqw2+1IJBJdBSh3d3cRDAb1yTmXyzXT46ZpGq6urp49zn6zg47T1tYWwuHwzNpBEOsIibQEQSwESZJWapkesfqYyXtjAwpFUZYyYKJldAQxOaIoIhgM4u7ubiGf17vkmQ1gmfBirHbO8gw1TTNV3MkMmqbh7OxMd2UyUQR4EvkODw9xfX2NbDY7clv39/eo1WrY2dkZ6cjkOA4+nw8+nw/tdhu5XA6Pj49dS4H7OSyNQu0sRDVg/CX+qwDHcYjFYqjX6ygUCn1fY1z6bhQCR000np+f4+joaKIYJUEQEI/H8fj4+MwNOujaNMxJy9pqzDNeZTFymJN1Vo5hth+Nhe9esghoLPI3DBYhwYRboLt+gFFAn/b8Ydvql3UMkDDLYP00u84pigJRFBEIBLC1tdX3PR6PZ27taTQaz1YgAMDGxoY+Qfv4+Njleo7FYiTQEgSA9btTIghibaHsTWIcem/s2fJSdhPKbv5lWV7aDXo+n4emaUttA0GsM8zBaXRwLQpjsSPgSWRgIhXrW4yi1bR0Op2uQWtvJiDHcdje3sbu7q4ph365XMbHjx9RLpdNt0GSJMTjcbx7904vaMYcgkw0tlgsEARBF6SYGDOpQMXESkEQUK/XUSwWx97GIFi7mSghy/LcljoHg8Ghz/fbN+z8HiSaMuF+0gnsbDarO8QlSdLdcDzPmz5e7LW9gieb/JzHUuNJr5csT5PdBwzb/jziFNi5tarC9aJgv7t+x5H1F+z8sVgsEEXR9KQ263+MfRP7bSuKom+X/Xvtk+Wsb2UTjGyfJJNJHB4eYmNjYynt6hdpIwgCwuEwBEFALBbDT3/6U+zu7sLpdCIQCCCRSCyhpQSxepCTliCIhUEFt4hxMGbJsaWkxsH3KtyYs2XHkiRBURTEYrFlN4kg1gqj89AoZrH/LtvJNyqHdhwEQYDH40G5XIbNZoPdbu/7ukAgAKfTiS9fvozMHVUUBZ8/f+7a1vb2dtey1n5YLBYcHR09Wy5vsVgGFpRhziyzS+GZSNjrdEun0/D5fKa20Uu73UalUkG1WkW1Wh0Yo2Sz2RAKhRAMBmcmMk7qqGb7S5IkXXAy0ul0cHZ2hsPDw4HnxCDa7TZEUcT29jbcbjc0TcN3332nTy6wAkq9vyugewn/MEFzHr8/JiQx1/oo2CQtE+tYNMci7gOYCL7MVTvrjFFM7Z18Mrqh+zm3V+E+b5XoV+SxX3b44eEh3G73opvXBRPbje2LRqNdk7EcxyEQCCAQCJDRgSAMkEhLEMTCWMdljsTyYJVc+w1qlw3P87Db7dA0DfV6HTzPIxAILLtZBLF2GIWvfr/zZQ7SQ6EQNjc3Z5o7HYvF0G63R64ssVqtOD4+Rjabxd3dXde+qVar+PDhA25vb7G5uYn379/rz43jSBYEAT6fr0ukHVeAYsJLP6GPuXB7j2G1WsXd3R2CwaDpAmLtdhupVMp0cbVms4nb21ukUikEAgGEQqGpJopVVUUmk9EFXyaWjCMUsmsZ219MlGDZqZ8/f8bR0ZGp/GNGKBRCo9HoyrVNJBJ6FqSiKJAkSZ/g7BV4zBzveYknZmI0mDjLnNIAngnP84RNEJuNASAGM2jCzZgRTgyG/VaYmM2KgvXut2g0unSBlpFIJLriX4ZFGZAgTxA/wq3SrMXPf/5z7be//e2ym0EQxJzodDr4h3/4h2U3gyBMw/M83G43JEnSl5OyarN0Q0kQ09PpdPD73/++7+Dd6J5bFKIowmazIRKJTOz2HIXROWaGTqeDdDqNVquFfD6Pv/zLv0SxWNRz3r1eL7755htdhGQZtKFQCG63e2hflUql8PDwoP89LEOVidX9XM7sWPV+z1GZrKFQCBsbG2g0Gl3LnNlydlYcqzd3dRLsdju8Xi88Ho/pIjntdhuZTAaZTKbvOcqyMtlz7G/jPmLfgzkyO53OwMxOURTx5s2bsVy7pVLpWfGxQqGAy8vLrtUoRoztMgNrb+82phUujQXX+h3f3uJfxvexiVzje2clpC6igBpBmGXQ76Af8Xh8aREH/ajX66hWq3pG7c7OzlgTUQTxkuA47ltN034+6nXkpCUIYmEMWkJJEKuI3W6H3W6HoiioVCrgOA67u7tjL0clCGIwFosFNput77L+UQLfrHC73djY2IDNZptL/ibw5OxsNBpQVRWpVAp2ux0HBwemhEKLxaIPuv/whz9AVVVsbGxA0zR4vV7c39/jw4cP+MUvfgHgSbAqFAooFAqwWq0Ih8MIhUJ9HcG912UmrDIhkeM43bU4SCTgeR4OhwOiKKLRaECWZT2L0nj8WA4hWzrOjv3t7W1fh6xxOf4sYCLBw8MDeJ6Hy+WC2+1Gp9NBq9VCq9WCLMuwWCyQJAkcx6FcLg8Vh1nbRFFEu93uaqux4jzbT+z5QREDsizj8vISh4eHpicCPR5Pl2AJAH6/H4qi4Pr6uqsNkzKovdP+Rvu9l+2nfttm5yR7vLdNbJ+PK9b2VpwfFQFBEGYRRRGCIPQtojUKo5N7HWm1Wjg5Oen6LWWzWWxubi6xVQSx+pBISxD
```

Comment:
it might be good to show here how to use the `geometry` column of the GeoDataFrame (as you won't have this Latitude / Longitude column if you read eg a Shapefile)